### ISSUE TITLE
Heightmap feature is broken (this is a raspberry related issue!)

### ISSUE THREAD

**Issue by simonCor** [pihnat/rpi-Candle#1]
It seems, that the raspberry pi has an issue only happening on the rpi.

Test on a rpi4 (did the same on an x86 linux system - no problem there).

The heightmap feature always returns an error 22 from grbl when Probe is executed. AFAIK error 22 means that the feedrate was not set.

How reproduce:
- Load an .nc file
- Click on Create in Heightmap drawer
- Click on Auto in Border  (other values do not matter)
- Click on Probe
- CNC starts for a second then the error is shown

Screenshot:
![IMG_20200405_100513457_HDR](https://user-images.githubusercontent.com/1113780/78469920-a7be8e00-7725-11ea-9fd8-a9908fd1d177.jpg)

**Comment by simonCor** [comment]
Solved it: It seems to be a problem with the default settings.

In the settings is setting for the feedrate when creating the heightmap - this is set to 0.

**Comment by pihnat** [comment]
Hi Simon,
Glad to hear it was easy to fix. I'd be surprised if the raspi version didn't behave the same as the Linux version. BTW after I built the raspi version of Candle in qt I noticed there was no **settings.ini** file (of course). When Candle is run the first time it creates the file with default settings. For example all the colours in the 3D window are black. So I created a settings file which has some reasonable colours, a serial port with the grbl baud rate of 115200, etc and included it in the Candle_x_x_x.zip file.
